### PR TITLE
[ADD] doc: multi-company howtos

### PR DESCRIPTION
**Pull request by Feyensv**
Add tutorials/examples on multi-company features of the ORM.

In master, will also contain information about new with_company feature.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/howtos/company.rst [hunk 1]
```diff
@@ -2,224 +2,181 @@
...
-  </record>
+   <record model="ir.ui.view" id="record_form_view">
+       <field name="name">record.restricted.form</field>
+       <field name="model">record.restricted</field>
+       <field name="arch" type="xml">
+           <form>
+               <sheet>
+                   <group>
+                       <group>
+                           <field name="company_id" groups="base.group_multi_company"/>
```
Comment (Feyensv) [inline doc/howtos/company.rst]: @AntoineVDV it's a bit sad to have a view with a only a poor little sad field :sob:

Comment (AntoineVDV) [reply doc/howtos/company.rst]: Then add back some other fields but they should clearly be meaningless, like `name` or such.

**Comment by Feyensv** [comment]
robodoo delegate=AntoineVDV

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**